base/fabric8-analytics-server/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM registry.devshift.net/osio-prod/base/pcp:latest
+
+COPY hack/_copr_jpopelka-mercator.repo /etc/yum.repos.d/
+
+# python3-pycurl is needed for Amazon SQS (boto lib), we need CentOS' rpm - installing it from pip results in NSS errors
+RUN yum-config-manager --enablerepo=rhel-7-server-optional-rpms && \
```

Comment:
For what package is this required?

Reply:
mercator requires maven which requires this

Reply:
:+1: